.github/workflows/add-to-project.yaml

```diff
@@ -15,5 +15,5 @@ jobs:
     steps:
       - uses: actions/add-to-project@v0.4.0
         with:
-          project-url: https://github.com/orgs/ROCmSoftwarePlatform/projects/26
+          project-url: https://github.com/orgs/ROCmSoftwarePlatform/projects/20
```

Comment:
Is this change necessary ? undo this change otherwise.

Reply:
merge error; fixed